_po/ja/how-to/laravel.po

```diff
@@ -639,33 +972,45 @@ msgid ""
 "</x-guest-layout>\n"
 "```"
 msgstr ""
+"                </div>\n"
+"            </div>\n"
+"        </div>\n"
+"    </div>\n"
+"</x-guest-layout>\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.